xml/art_solution_automation.xml

```diff
@@ -0,0 +1,438 @@
+            by &suse;.
+          </para>
+        </listitem>
+      </varlistentry>
+    </variablelist>
+  </sect1>
+
+  <sect1 xml:id="sec-sol-autom-using-salt-formulas">
+    <title>Using &salt; Formulas</title>
+    <remark>toms 2020-06-29: Is there a general way to install/use these? Should
+    we tell our readers about that?</remark>
+    <para>
+      To use &salt; formulas from &suse;, execute the following steps:
+    </para>
+    <!--
+    <procedure>
+      <step>
+        <para>Install the </para>
+        <screen>git clone https://github.com/SUSE/shaptools.git
+cd shaptools
+sudo python setup.py install</screen>
+      </step>
+      <step>
+        <para></para>
+      </step>
+    </procedure>
+    -->
+  </sect1>
+
+  <sect1 xml:id="sec-sol-autom-hana-deploy">
+    <title>&hana; Automated Deployment</title>
+    <para>
+      &salt; formula <package>saphanabootstrap-formula</package> for bootstrapping
+      and managing &hana; platform and system replication.
+    </para>
+    <formalpara>
+      <title>Link:</title>
+      <para>See <link xlink:href="https://github.com/SUSE/saphanabootstrap-formula"/>.</para>
+    </formalpara>
+    <itemizedlist>
+      <title>Capabilities for <package>saphanabootstrap-formula</package></title>
+      <listitem>
+        <para>
+          Installs &hana;.
+        </para>
+      </listitem>
+      <listitem>
+        <para>
+          Configures system replication.
+        </para>
+      </listitem>
+      <listitem>
+        <para>
+          Configures &ha; cluster.
+        </para>
+      </listitem>
+      <listitem>
+        <para>
+          Configures optional monitor exporter.
+        </para>
+      </listitem>
+    </itemizedlist>
+    <para>
+      The &salt; formula can be used in these scenarios:
+    </para>
+    <itemizedlist>
+      <title>Scenarios for <package>saphanabootstrap-formula</package></title>
+      <listitem>
+        <para>
+          &hana; single instance automated deployment.
+        </para>
+      </listitem>
+      <listitem>
+        <para>
+          Two &hana; instances on system replication setup.
+        </para>
+      </listitem>
+      <listitem>
+        <para>
+          Two &hana; instances on system replication and cluster scale-up
+          performance-optimized configuration.
+        </para>
+      </listitem>
+      <listitem>
+        <para>
+          Two &hana; instances on system replication setup, cluster scale-up
+          cost-optimized configuration, and extra &hana; installation on a secondary host.
+        </para>
+      </listitem>
+      <listitem>
+        <para>
+          Cloud Service Provides (Azure, AWS, and GCP) automated adjustments for each provider 
+          as best-practice.
+        </para>
+      </listitem>
+    </itemizedlist>
+    <example>
+      <title>Installation</title>
+      <screen>zypper addrepo --refresh obs://network:ha-clustering:Factory ha-clustering
```

Comment:
This obs repository is deprecated. The development repository is: https://build.opensuse.org/project/show/network:ha-clustering:sap-deployments:devel

And these packages are shipped to `openSUSE:Factory` too

Reply:
Ok, thanks! :+1: I've corrected that.

However, I've extracted this information from the README. In that case, the README file needs also an update. :wink:

Reply:
Ok. Yes, as commented somewhere else, if we are going to use the github repos as the main source of documentation, we need to check them completely and rewrite some parts